### ISSUE TITLE
No no-js→js swap if Add CSS Classes is unselected

### ISSUE THREAD

**Issue by joan** [Modernizr/Modernizr#1826]
Source says
```// Change `no-js` to `js` (independently of the `enableClasses` option)```
But this code is not present in build if Add CSS Classes is unselected (and `no-js` isn't swapped to `js`).

**Comment by patrickkettner** [comment]
hey @joan!
I am not clear on what you mean - could you rephrase?

thanks!

**Comment by joan** [comment]
Sorry, I edited!

**Comment by thasmo** [comment]
I've got this:
```js
 _config: {
  'classPrefix' : '',
  'enableClasses' : true,
  'enableJSClass' : true,
  'usePrefixes' : true
},
```
and it does *not* set classes on the HTML element at all, neither does it replace `no-js`.

That's the URL I use for installing Moderniz with bower:
https://modernizr.com/download?touchevents-dontmin-setClasses-.zip

**Comment by thasmo** [comment]
It seems there are functions missing when download via Bower.

**Comment by joan** [comment]
I tried with https://modernizr.com/download/

**Comment by thasmo** [comment]
@patrickkettner, any chance you have a look at this? Installation using the download-url doesn't seem to work for some cases it seems.

**Comment by patrickkettner** [comment]
@thasmo the issue with your URL is that you are using `setClasses` rather than `setclasses`.  Using the lowercased version makes it work. I opened an issue for this on the modernizr-neue side

**Comment by patrickkettner** [comment]
the issue that @joan brought up is because `setClasses` functionality used to be to always include the ability to swap `no-js` -> `js`. This was later modified to make that swap under a flag (that defaulted to `true`). No one ever has mentioned wanting the removal without classes as well. So basically [this chunk](https://github.com/Modernizr/Modernizr/blob/d5beea6e822ef9cdfee7e094d8d0e5e15dfc8801/src/setClasses.js#L22-L25) should be moved into a separate `src` file, just like everything else
